### ISSUE TITLE
Проблемы при использовании localrunner совместно с отлидчиком

### ISSUE THREAD

**Issue by dbf256** [MailRuChamps/raic#19]
1. При отладке и переключении между ide и localrunner в нем пропадает изображение и начинают показываться всякие артефакты, вроде куска IDE, рабочего стола

![image](https://user-images.githubusercontent.com/2550295/50136332-89225280-02a8-11e9-9620-725cf88cd2d7.png)

2. Во время отладки после прохождения какого-то таймаута игра отцепляется от отладчика и продолжается до забитого гола.

**Comment by SannikovDmitry** [comment]
Без отладчика все нормально работает?

**Comment by dbf256** [comment]
@SannikovDmitry да,  такое впечатление, что когда стоит на паузе, то при переключении между окнами визуализатор не перерисовывает картинку, вот и поверх оказываются куски других окон.

**Comment by tyamgin** [comment]
+1
Невозможно отлаживать.
Когда стою на точке останова в Clion, окно local_runner'a в находится подвешенном состоянии.
В прошлых RAIC'ах такой проблемы не было.

**Comment by kuviman** [comment]
Попробуйте скачать новую версию раннера (0.3.1). Теперь во время дебага должно все работать

**Comment by dbf256** [comment]
@kuviman попробовал, с отрисовкой проблемы нет, единственное, добавился небольшой артефакт - если постоять на отладчике, а потом уйти с него, то какое-то время резко возрастает резкость смены кадров в localrunner, через какое-то время все возвращается к норме. Как будто там есть какой-то учет кадров в секунду и пока я стою на отладчике счетчик не двигается, а после снятия он пытается подогнать показ под этот параметр.
